packages/oas-to-snippet/README.md

```diff
@@ -24,41 +24,62 @@ import petstore from './petstore.json';
 const apiDefinition = new Oas(petstore);
 const operation = apiDefinition.operation('/pets', 'get');
 
-// This is a keyed object containing formData for your operation. Available keys are: path,
```

Comment:
Trimmed these Markdown code snippet lines down to 100 character lengths because 120 looks like shit on small screens on NPM and Github.

Reply:
fwiw i don't think this is a huge deal since both npm and github support side-scrolling on these code samples and even 100 characters is still too wide for certain screens like mobile etc.